src/style.css

```diff
@@ -1 +1,25 @@
 /* add styles here */
+body {
+  margin: 0;
+}
```

Comment:
Avoid styling with tag names (except for `html` and `body`). If you need to apply styles for specific elements, use classes instead.